<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1490</width>
    <height>645</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>-1</y>
      <width>1491</width>
      <height>591</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="3,2,7">
     <property name="leftMargin">
      <number>7</number>
     </property>
     <property name="topMargin">
      <number>7</number>
     </property>
     <property name="rightMargin">
      <number>7</number>
     </property>
     <property name="bottomMargin">
      <number>7</number>
     </property>
     <item>
      <layout class="QGridLayout" name="gridLayout" columnstretch="3,0">
       <item row="0" column="0" colspan="2">
        <widget class="QWebEngineView" name="webEngineView" native="true">
         <property name="url" stdset="0">
          <url>
           <string>about:blank</string>
          </url>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QComboBox" name="comboBox"/>
       </item>
       <item row="2" column="0">
        <widget class="QComboBox" name="comboBox_2"/>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="datetime">
         <property name="text">
          <string>날짜/시간</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="locselection">
         <property name="text">
          <string>측정소 선택</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="2" column="1">
        <widget class="QPushButton" name="refilebutton">
         <property name="text">
          <string>변환하기</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QTableView" name="tableView"/>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="import_2">
         <property name="text">
          <string>불러오기</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QPushButton" name="analysisButton">
         <property name="text">
          <string>분석하기</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QPushButton" name="showdfButton">
         <property name="text">
          <string>표 출력</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="2" column="2">
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>PushButton</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="comboBox_4"/>
       </item>
       <item row="2" column="0">
        <widget class="QComboBox" name="comboBox_3"/>
       </item>
       <item row="0" column="3">
        <widget class="matplotlibWidget" name="CO" native="true"/>
       </item>
       <item row="0" column="0">
        <widget class="matplotlibWidget" name="SO2" native="true"/>
       </item>
       <item row="1" column="1">
        <widget class="matplotlibWidget" name="PM25" native="true"/>
       </item>
       <item row="1" column="0">
        <widget class="matplotlibWidget" name="PM10" native="true"/>
       </item>
       <item row="0" column="2">
        <widget class="matplotlibWidget" name="O3" native="true"/>
       </item>
       <item row="1" column="2">
        <widget class="matplotlibWidget" name="NOX" native="true"/>
       </item>
       <item row="0" column="1">
        <widget class="matplotlibWidget" name="NO2" native="true"/>
       </item>
       <item row="1" column="3">
        <widget class="matplotlibWidget" name="NO" native="true"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1490</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebEngineView</class>
   <extends>QWidget</extends>
   <header location="global">QtWebEngineWidgets/QWebEngineView</header>
  </customwidget>
  <customwidget>
   <class>matplotlibWidget</class>
   <extends>QWidget</extends>
   <header>matplotlibwidgetFile.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
